packages/terra-alert/src/Alert.jsx

```diff
@@ -162,26 +144,20 @@ const Alert = ({
   );
 
   return (
-    <ResponsiveElement
-      tiny={(
-        <div {...attributes} className={narrowAlertClassNames}>
-          <div className={bodyClassNameForNarrowParent}>
-            {getAlertIcon(type, customIcon)}
-            {alertMessageContent}
-          </div>
-          {actionsSection}
-        </div>
-      )}
-      small={(
-        <div {...attributes} className={wideAlertClassNames}>
-          <div className={cx(['body', 'body-std'])}>
+    <ResponsiveElement onChange={value => setBreakpoint(value)}>
```

Comment:
Looking at the styles in the style sheet, it seems we only care about flipping between a narrow and wide Alert.

Can we add a check in the onChange callback we provide to only update the state when the breakpoint flips from narrow to large? 

If we do this, we shouldn't need to  add new styles, but use the existing ones.

```jsx
const [isNarrow, setIsNarrow] = useState();
const alertClassNames = cx([
    'alert-base',
    type,
    { 'narrow': isNarrow }
    { 'wide': !isNarrow }
    attributes.className,
    { [`${customColorClass}`]: type === AlertTypes.CUSTOM },
  ]);

const bodyClassNameForParent = cx([
    'body',
    'body-std'
   { 'body-std': !isNarrow || (isNarrow && !dismiss && !action}
   { 'body-narrow': isNarrow && (dismiss || action) }
  ]);

<ResponsiveElement
    onChange={(value) => {
         const showNarrowAlert = value === 'tiny';
         if (showNarrowAlert !== isNarrow ) {
           setBreakpoint(showNarrowAlert);
         }
     }
>
      <div {...attributes} className={alertClassNames}>
          <div className={bodyClassNameForNarrowParent}>
            {getAlertIcon(type, customIcon)}
            {alertMessageContent}
          </div>
          {actionsSection}
        </div>
</ ResponsiveElement>
```